### DISCUSSION TITLE
Please clearify WixBundleSourceProcessFolder

### DISCUSSION THREAD

**Discussion by cz-dev-ge** [wixtoolset/issues#6831] [Questions] [answered]
Hi there,

I don't understand what the explanations mean:

+ WixBundleOriginalSource - gets the source path from where the bundle originally ran.
+ WixBundleOriginalSourceFolder - gets the folder from where the bundle originally ran.
+ WixBundleSourceProcessPath - gets the source path of the bundle where originally executed. Will only be set when bundle is executing in the clean room.
+ WixBundleSourceProcessFolder - gets the source folder of the bundle where originally executed. Will only be set when bundle is executing in the clean room.

Especially:

+ aren't run and executed synonyms?
+ both say "originally" - is that intended?
+ what is the "clean room" - under which circumstances is this not set?

Best regards,
Georg

**Comment by rseanhall** [comment, marked as answer]
https://github.com/wixtoolset/wix3/pull/544
